openbb_terminal/econometrics/econometrics_controller.py

```diff
@@ -181,7 +181,6 @@ def __init__(self, queue: List[str] = None):
                 "root",
                 "granger",
                 "coint",
-                "regressions",
```

Comment:
Is this by accident?

Reply:
Nop, it's the 2nd fix. We don't have any 'regressions' cmd